### PR TITLE
Updated Turkish Language

### PR DESCRIPTION
**Pull request by fr0stb1rd**
## Description

Updated Turkish Language

## Type of change

- [x] Improvement (refactoring and improving code)

## How has this been tested

No need. only strings changed.

### CODE REVIEW

**Review by cotes2020 — COMMENTED** [review]
Is there any good reason for this change?

**Comment by fr0stb1rd** [comment]
> Is there any good reason for this change?

spelling mistakes corrected.

File: _data/locales/tr-TR.yml [hunk 1]
```diff
@@ -70,8 +70,21 @@ post:
...
       succeed: Link kopyalandı.
+# Date time format.
+# See: <http://strftime.net/>, <https://day.js.org/docs/en/display/format>
+df:
+  post:
+    strftime: "%b %e, %Y"
+    dayjs: "ll"
+  archives:
+    strftime: "%b"
+    dayjs: "MMM"
```
Comment (cotes2020) [inline _data/locales/tr-TR.yml:81]: These cause elements to jitter on load, so remove them.

```suggestion
```

File: _data/locales/tr-TR.yml [hunk 2]
```diff
@@ -70,8 +70,21 @@ post:
...
 # categories page
 categories:
-  category_measure: kategori
-  post_measure: gönderi
+  category_measure:
+    singular: kategori
+    plural: kategori
+  post_measure:
+    singular: gönderi
+    plural: gönderi
```
Comment (cotes2020) [inline _data/locales/tr-TR.yml:90]: When the singular and plural are the same, there is no need to repeat the definition, restore them.

```suggestion
  category_measure: kategori
  post_measure: gönderi
```

**Review by cotes2020 — APPROVED** [review]

**Pull request merged**